great_expectations/rule_based_profiler/helpers/util.py

```diff
@@ -1050,8 +1051,28 @@ def get_or_create_expectation_suite(
     return expectation_suite
 
 
-def sanitize_parameter_name(name: str) -> str:
+def sanitize_parameter_name(
+    name: str,
+    suffix: Optional[str] = None,
+) -> str:
     """
-    This method provides display-friendly version of "name" argument.
+    This method provides display-friendly version of "name" argument (with optional "suffix" argument).
+
+    In most situations, "suffix" is not needed.  However, in certain use cases, "name" argument is same among different
+    calling structures, whereby "name" can be disambiguated by addition of distinguishing "suffix" argument.  Using
+    list of "MetricConfiguration" objects as example, "metric_name" and "metric_domain_kwargs" are commoonly same among
```

Comment:
"commoonly" -> "commonly"

Reply:
@NathanFarmer Fixed -- nice catch -- thank you!